### DISCUSSION TITLE
suricata user

### DISCUSSION THREAD

**Discussion by tommorgan365** [Security-Onion-Solutions/securityonion#2266] [Unsupported Versions] [answered]
installing new sensor using the github repository Centos 7 image.
Ran into error at the end of the install with suricata and crontab.

Does not look like suricata user is pointing to right home directory?
$ sudo su - suricata
Last login: Tue Dec  8 22:00:11 UTC 2020 on pts/0
su: warning: cannot change directory to /opt/so/conf/suricata: Permission denied
-bash: /opt/so/conf/suricata/.bash_profile: Permission denied
-bash-4.2$ crontab -l
You (suricata) are not allowed to use this program (crontab)
See crontab(1) for more information
-bash-4.2$

**Comment by tommorgan365** [comment]
Just installed second sensor - same issue.

**Comment by dougburks** [comment, marked as answer]
> installing new sensor using the github repository Centos 7 image.

To clarify, are you using our Security Onion ISO image or a standard CentOS 7 ISO image?

> Ran into error at the end of the install with suricata and crontab.

What exactly was the error?

**Reply by tommorgan365** [reply]
Standard Centos 7 ISO -- hardened.  We had to add the suricata user to /etc/cron.allow.  The error message was You (suricata) are not allowed to use this program (crontab) -- at the end of of the install log.

so this is is closed.
